File: benchmarks/accuracy/fixtures/java/interprocedural_sql_injection.java
import java.sql.*;
import javax.servlet.http.*;

public class UserController extends HttpServlet {
    private Connection conn;

    private ResultSet findUser(String name) throws SQLException {
        String sql = "SELECT * FROM users WHERE name = '" + name + "'";
        Statement stmt = conn.createStatement();
        return stmt.executeQuery(sql);
    }

    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws SQLException {
        String name = request.getParameter("name");
        findUser(name);
    }
}

File: benchmarks/accuracy/fixtures/python/interprocedural_sql_injection.py
import sqlite3
from flask import Flask, request

app = Flask(__name__)


def find_user(conn, name):
    query = f"SELECT * FROM users WHERE name = '{name}'"
    return conn.execute(query).fetchall()


@app.route("/user")
def user():
    conn = sqlite3.connect("app.db")
    name = request.args.get("name")
    return str(find_user(conn, name))
